k8s/iabweb/cron.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cron-weekly
+spec:
+  schedule: "30 0 * * 0" # "At 00:30 on Sunday."
+  timeZone: "Etc/UTC"
+  concurrencyPolicy: Forbid # No concurrent runs
+  jobTemplate:
+    spec:
+      backoffLimit: 0 # No retries
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: cron-weekly
+              image: "ghcr.io/ietf-tools/www:$APP_IMAGE_TAG"
```

Comment:
How will this impact when deployment is done with the `latest` tag?
Is this going to fetch the `latest` image every time?
If so that can lead to issues.
(cc @NGPixel)

Reply:
I think it's best not to use the `latest` tag. Instead, always specify a version.

The issue you described re: `latest` may not be specific to cron jobs. What happens if the `wagtail` pod gets deleted (to force pod recreation for instance) - will it fetch the latest `latest` image available then?

Reply:
Yes, discussed elsewhere.